index.html

```diff
@@ -49,21 +50,33 @@
 
     const volume = Math.min(1, Math.abs(beta) / 90);
     const normPitch = Math.max(0, Math.min(1, (gamma + 90) / 180));
-    const pitchIndex = Math.floor(normPitch * notes.length);
     const pitch = 440 * Math.pow(2, normPitch);
+    var min = Number.MAX_VALUE;
+    var note;
+    var pitchIndex;
+    var diff;
+    for (var i = 0; i < notes.length; i++) {
```

Comment:
Can we make this optional with e.g. an on/off switch in the UI? I like how crazy the continuous sound is, but also want discretized sometimes 😉